### ISSUE TITLE
Error deleting menu

### ISSUE THREAD

**Issue by djordje41** [mattmilburn/strapi-plugin-menus#150]
When trying to delete simple menu I just created, I get this error in console:
![image](https://github.com/mattmilburn/strapi-plugin-menus/assets/110818470/620447bc-affd-4cd5-b6e7-7807f522f9c4)
No request has been sent to the server.

I believe it has to do with fetchClient we are using for performing strapi requests. In the useFetchClient.d.ts we have this:
```
declare const useFetchClient: () => {
    get: <TData = any, R = import("axios").AxiosResponse<TData, any>, TSend = any>(url: string, config?: import("axios").AxiosRequestConfig<TSend> | undefined) => Promise<R>;
    put: <TData_1 = any, R_1 = import("axios").AxiosResponse<TData_1, any>, TSend_1 = any>(url: string, data?: TSend_1 | undefined, config?: import("axios").AxiosRequestConfig<TSend_1> | undefined) => Promise<R_1>;
    post: <TData_2 = any, R_2 = import("axios").AxiosResponse<TData_2, any>, TSend_2 = any>(url: string, data?: TSend_2 | undefined, config?: import("axios").AxiosRequestConfig<TSend_2> | undefined) => Promise<R_2>;
    del: <TData_3 = any, R_3 = import("axios").AxiosResponse<TData_3, any>, TSend_3 = any>(url: string, config?: import("axios").AxiosRequestConfig<TSend_3> | undefined) => Promise<R_3>;
};
export { useFetchClient };
//# sourceMappingURL=useFetchClient.d.ts.map
```

But we are trying to do this: `fetchClient.delete(getRequestUrl(id))` (admin/src/pages/IndexView/index.js: line 97). I changed `delete` to `del` and it worked locally.

**Comment by mattmilburn** [comment]
Hi @djordje41 Thank you so much for pointing this out! I'll make a PR for this quick fix this and I'll be deploying a new patch version of the plugin today 👍🏻

**Comment by djordje41** [comment]
You are welcome! Thank you for a rapid response!

**Comment by mattmilburn** [comment]
Hi again @djordje41 This bug is now fixed in the latest [version 1.6.1](https://github.com/mattmilburn/strapi-plugin-menus/releases/tag/v1.6.1) of the plugin 🎉
